src/shared/components/home/setup.tsx

```diff
@@ -73,6 +73,7 @@ export class Setup extends Component<any, State> {
                 onSaveSite={this.handleCreateSite}
                 siteRes={this.state.siteRes}
                 themeList={this.state.themeList}
+                loading={false}
```

Comment:
This can always be `false` since the page is redirected upon submit of this form.